src/ng-select/ng-select.component.html

```diff
@@ -43,12 +62,22 @@
 
     <div class="ng-spinner-loader" *ngIf="loading"></div>
 
-    <span *ngIf="showClear()" class="ng-clear-wrapper" title="{{clearAllText}}">
-        <span class="ng-clear" aria-hidden="true">×</span>
+    <span *ngIf="showClear()" class="ng-clear-wrapper">
+        <button class="ng-icon-button ng-clear-button"
```

Comment:
Probably we can avoid the button on the chevron icon, but how can a disabled user clear the widget value in one go, especially for a multi-select configuration?

Reply:
@brunano21 - I'd agree that having Clear be focusable would be a great feature but would also agree that the chevron being focusable could just create extra layers of unnecessary tabbing since regular keyboard controls function for opening/close/selecting items. The backspace to delete is nice, but being able to focus + clear all is a huge win for keyboard users.

Great work on this PR!